packages/manager/modules/pci/src/projects/project/storages/databases/database/topics/create-topic/create-topic.controller.js

```diff
@@ -0,0 +1,58 @@
+import get from 'lodash/get';
+import capitalize from 'lodash/capitalize';
+
+export default class {
+  /* @ngInject */
+  constructor($translate, DatabaseService) {
+    this.$translate = $translate;
+    this.capitalize = capitalize;
+    this.DatabaseService = DatabaseService;
+  }
+
+  $onInit() {
+    this.trackDashboard('topics::add', 'page');
+    this.model = {
+      name: '',
+      replication: 3,
```

Comment:
Move all these numbers to a constant file

Reply:
Done here: https://github.com/ovh/manager/pull/5488/commits/45d16da89ff2c70503a9a111c3fd5996a5eecbe1